Sources/MockoloFramework/Templates/ClassTemplate.swift

```diff
@@ -79,6 +80,11 @@ extension ClassModel {
         
         let extraInits = extraInitsIfNeeded(initParamCandidates: initParamCandidates, declaredInits: declaredInits,  acl: acl, declType: declType, overrides: metadata?.varTypes)
 
+        var inheritedTypesStr = ""
+        for inheritedType in inheritedTypes {
+            inheritedTypesStr += ", " + inheritedType
+        }
+
```

Comment:
Maybe this can be more beautify like this:
```swift
var inheritedTypes = inheritedTypes
inheritedTypes.insert("\(moduleDot)\(identifier)", at: 0)
```

```swift
\(acl)\(finalStr)class \(name): \(inheritedTypes.joined(", ")) {
```